cita-executor/core/src/libexecutor/block.rs

```diff
@@ -208,6 +206,9 @@ impl ExecutedBlock {
                 b
             });
 
+        block.set_state_root(state_root);
```

Comment:
It's nothing about the commit.
A new commit is better.

Actually it's not a good habit to do things beyond the pr's purpose.
I've done this before, too. Maybe we all should follow it and review this each other.

Reply:
BTW, I'll write the doc about the rule: PR guide, review guide, and something like this.